client/client-multi/client-c/src/kaa/utilities/kaa_aes_rsa.h

```diff
@@ -60,4 +62,20 @@ kaa_error_t aes_encrypt_decrypt(int mode, const uint8_t *input, size_t input_siz
 int rsa_sign(mbedtls_pk_context *pk, const uint8_t *input, size_t input_size,
         uint8_t *output, size_t *output_size);
 
+/**
+ * Encrypt data with pubcic RSA key.
+ *
+ * @param[in]   key         Public RSA key.
+ * @param[in]   key_size    Size of public RSA key.
+ * @param[in]   input       Input data.
+ * @param[in]   input_len   Size of input_data.
+ * @param[out]  output      Pointer to buffer where the encrypted data will be stored.
+ *
+ * @return The error code.
+ * @retval KAA_ERR_NONE Success
+ * @retval KAA_ERR_INVALID_PUBKEY The key is invalid or encryption has failed.
```

Comment:
Let's leave it as is for now.
I opened an issue to do this http://jira.kaaproject.org/browse/KAA-1430, please add TODO in implementation (see my comment)